### PR TITLE
remove-zero-baseline-consideration

### PR DESCRIPTION
**Pull request by moontrip**
This will resolve https://github.com/VEuPathDB/web-monorepo/issues/147. I have tested several cases, but one of the best and simplest way is to set the default value of the isNonZeroBaseline prop as true for. Actually, it is not used anywhere yet.

Here are various screenshots using year variable after applying this:

year variable 01

year variable 02

year variable 03

year variable 04

### CODE REVIEW

File: packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts [hunk 1]
```diff
@@ -24,7 +24,8 @@ export function useDefaultAxisRange(
   logScale?: boolean,
   axisRangeSpec = 'Full',
   // check non-zero baseline for continuous overlay variable
-  isNonZeroBaseline: boolean = false
+  // but not used in any Viz yet, so change the default to true
+  isNonZeroBaseline: boolean = true
```
Comment (dmfalke) [inline packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts:28]: I don't really understand this parameter. Can you add a comment explaining what "non-zero baseline" means? Thanks!

**Review by dmfalke — CHANGES_REQUESTED** [review]

File: packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts (hunk 1 above)
Comment (moontrip) [reply packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts]: @dmfalke That's a good question. The isNonZeroBaseline prop was intended to deal with setting a range of continuous variable for colormap legend: https://github.com/VEuPathDB/web-components/issues/454. Initially, the range was designed to be computed via the useDefaultAxisRange hook. However, I reminded myself that overall procedure was changed without using the hook, thus the prop is actually no longer used. Thus, I removed the prop and changed accordingly. To address the issue of year variable, I simply removed zero (0) case from the calculation of the min at numberDateDefaultAxisRange(). These changes are committed.

Thank you for inspiring me to recall this 👍

Comment (dmfalke) [reply packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts]: Thanks for the explanation! This looks great.

**Review by dmfalke — APPROVED** [review]

File: packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts (hunk 1 above)
Comment (moontrip) [reply packages/libs/eda/src/lib/core/hooks/computeDefaultAxisRange.ts]: @dmfalke my pleasure 😄 Thank you for your reviews!

**Comment by d-callan** [comment]
Does this now never force through 0? I'm not sure I'm reading it right..

**Comment by dmfalke** [comment]
> Does this now never force through 0? I'm not sure I'm reading it right..

I think you're right. It seems to me, this can be fixed by changing this line:
https://github.com/VEuPathDB/web-monorepo/blob/e8f4d51eb12dfc120a17d7fc84a803ad9a3c9314/packages/libs/eda/src/lib/core/utils/default-axis-range.ts#L34

to
```
defaults.displayRangeMin ?? 0,
```

**Pull request merged**